src/static/assets/css/markdown.css

```diff
@@ -0,0 +1,695 @@
+@font-face {
```

Comment:
I don't think we need to track changes to this file.  Effectively, it's our file that we've just cribbed from an external source.